README.md

```diff
@@ -49,10 +53,21 @@ deploy:
     [another extend directory]: false
   ignore_pattern:
     [folder]: regexp  # or you could specify the ignore_pattern under a certain directory
+
+# keys inside repo do not matter, and can be omitted for single repo with token:
```

Comment:
which keys can be omitted? `repo.url` & `repo.token`? Can you give an example config for single repo?

Reply:
It means the name of repo.

For multiple repos config, we have:

```yaml
deploy:
  repo:
    repo_A: <repository url>[,branch]
    repo_B:
      url: <repository url>
      branch: [branch]
      token: [$ENV_TOKEN]
```

But for config with single repo, which I think should be the most situation, we will get something like:

```yaml
deploy:
  repo:
    my_repo:
      url: <repository url>
      branch: [branch]
      token: [$ENV_TOKEN]
```

In this case, name of repo might be superfluous so we can omit it. Example config for single repo is followed in the next 4 lines.

The description might be misleading so I updated it.

Reply:
you mean for a single repo, `my_repo:` key can be omitted, like

``` yml
deploy:
  repo:
    url: <repository url>
    branch: [branch]
    token: [$ENV_TOKEN]
```

> it only works with the repo object, not string

so the following wouldn't work?

``` yml
deploy:
  repo: <repository url>[,branch]
```

I'm assuming the following also wouldn't work?

``` yml
deploy:
  repo: 
    my_repo: <repository url>[,branch]
```

Reply:
Both will work as they are kinds of existing config.

For the first case, check test case ['single repo, branch after url'](https://github.com/hexojs/hexo-deployer-git/pull/135/files#diff-154adb73f62d17669901a380c640c280R29).

For the following one, the omit of name is optional and it's fine to keep it. This case is updated to the unit test.